collector/main.go

```diff
@@ -3,15 +3,24 @@ package main
 import (
 	"github.com/Kindling-project/kindling/collector/application"
 	"log"
+	"fmt"
+
+	"github.com/sugary199/collector-version/core"
```

Comment:
Same as the `go.mod` file. I think this line could be replaced with `core/core.go` (before you rename that package).

Reply:
`version` is truly better. I'll change it if we still use this method.

I did want to introduce a package. I see that our package references are imported from the GitHub repository, so I use my warehouse temporarily since  `core` has not been merged into the formal kindling repository.

Reply:
Sorry for my incorrect suggestion. `github.com/sugary199/collector-version/core` could be replaced with `github.com/Kindling-project/kindling/collector/core` instead. You can see our module name in the first line of `go.mod` is `github.com/Kindling-project/kindling/collector` which means you could just import the packages that belong to the module with its module name beginning.

Reply:
1、I‘ve replaced 'core' with 'version'.
2、Comments are added in front of the commands and codes that need to be modified if they are merged.
The specific location is 
'//  "github.com/Kindling-project/kindling/collector/version"' in 'main.go'
'# docker run -it -v $PWD:/collector kindlingproject/kindling-collector-builder bash -c 'go build -o kindling-collector -ldflags="-X 'github.com/Kindling-project/kindling/collector/version.CodeVersion=$GitCommit'"'
' in 'collector-build.sh'

Is this right?

Reply:
1、I‘ve replaced `core` with `version`.
2、Comments are added in front of the commands and codes that need to be modified if they are merged.
The specific location is
`// "github.com/Kindling-project/kindling/collector/version"` in `main.go`
`# docker run -it -v $PWD:/collector kindlingproject/kindling-collector-builder bash -c 'go build -o kindling-collector -ldflags="-X 'github.com/Kindling-project/kindling/collector/version.CodeVersion=$GitCommit'"`
in `collector-build.sh`

Is this right?